plaidcloud/rpc/connection/jsonrpc.py

```diff
@@ -182,7 +182,7 @@ def __init__(self, token, uri=None, verify_ssl=None, workspace=None, proxies=Non
 
         def call_rpc(method_path, params, fire_and_forget=False):
             response = http_json_rpc(
-                token, uri, verify_ssl,
+                token, f"{uri}{method_path}", verify_ssl,
```

Comment:
I *think* this'll be ok, but I wonder if it should use urljoin to be on the safe side

Reply:
urljoin wouldn't work in this case because the uri already contains a path. So doing `urljoin(uri, method_path)` would remove `json-rpc` from the url.

Reply:
![image](https://user-images.githubusercontent.com/45275736/122786573-5b208d80-d2ac-11eb-9d19-d8d39cc5461a.png)

`urljoin` works in my console

Reply:
That works for me too. The documentation sure makes it looks like it would remove the `json-rpc` component. I'll swap that in then.